webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -94,29 +95,76 @@ export const unselectRow = id => ({
   payload: id,
 });
 
-export const actionSelected = (actionType, selected, url) => async dispatch => {
+export const bulkResumeRequest = resumeTasks => {
+  const ids = resumeTasks.map(task => task.id);
+  const url = new URI('/foreman_tasks/api/tasks/bulk_resume');
+  url.setSearch('task_ids[]', ids);
+  return API.post(url);
+};
+
+export const bulkResume = (selected, currentURL) => async dispatch => {
+  const resumeTasks = selected.filter(task => task.isResumeble);
```

Comment:
```suggestion
  const resumeTasks = selected.filter(task => task.isResumable);
```
Correct spelling (sorry, you'll also have to change it everywhere else)

Reply:
English is fun

Reply:
I still think the `resumable` spelling should be fixed (sorry :confused:)